packages/select/src/components/query-list/queryList.tsx

```diff
@@ -146,6 +162,22 @@ export class QueryList<T> extends React.Component<IQueryListProps<T>, IQueryList
         return QueryList as new (props: IQueryListProps<T>) => QueryList<T>;
     }
 
+    /**
+     * Helper method for rendering `props.filteredItems`, with optional support for `noResults`
+     * (when filtered items is empty) and `initialContent` (when query is empty).
+     */
+    public static renderFilteredItems(
```

Comment:
Can we broaden the scope of this helper? How about instead of taking IItemListRendererProps, we take items: T[]; then an itemListRenderer can call it with either filteredItems or items, depending on which is more appropriate. Also, since this is meant to be called first and foremost by an itemListRenderer, it's a bit awkward that it's defined on QueryList, which we usually don't import directly when consuming Select and friends. Can we maybe make it a top-level import of the select package?

Reply:
cool yes def down for the root export.

taking the props as a first argument was a quick way to get off the ground, cuz it receives both items and `renderItem` in one go. the name indicates which set of items it'll use. my goal here was to streamline the 99% case; if you want to render `items` then you're on your own.

Reply:
Yep, fair enough on the streamlining part.